exercises/C-variables/exercise.js

```diff
@@ -1,3 +1,4 @@
 // Start by creating a variable `greeting`
-
-console.log(greeting);
+let greeting = new greeting;
+let greeting1 ="hin";
```

Comment:
good to see you using `let` instead of `var`, a small improvement would be to use `const` instead (as you never redefine the variable). Also, you don't need to use the `new` keyword here (that is used for other things we'll discuss at a later point). Here, just the `let greeting = 'hin'` would be fine.